pom.xml

```diff
@@ -57,6 +58,12 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
```

Comment:
test: add `mockito-inline` dependency to create inline mocks. Which is useful to mock `final` classes and methods.